src/tests/ftest/container/async.py

```diff
@@ -45,7 +45,7 @@ def test_create_async(self):
         :avocado: tags=cont_create_async,test_create_async
         """
         self.add_pool()
-        pool_handle = self.pool.pool.handle
+        ph = self.pool.pool.handle
```

Comment:
I'm curious about this change.  I was under the impression that more explicit variable names are preferred in python.

Reply:
You're looking at one commit from the series only.

I changed ph to pool_handle for the reasons you said but then this test failed in CI so I reverted changes to this file so the PR could move forward but it turns out this test is failing anyway and my changes were not the cause.  This PR no longer updates all the container tests and the description says but leaves this failing one untouched.